lib/validation.rs

```diff
@@ -100,7 +100,11 @@ pub(crate) fn validate_get_era_info_response(
                 .map_err(|_| ValidateResponseError::ValidateResponseFailedToParse)?;
             let proofs: Vec<TrieMerkleProof<Key, StoredValue>> =
                 bytesrepr::deserialize(proof_bytes)?;
-            let key = Key::EraInfo(era_id);
+            let key = if result.api_version < ProtocolVersion::from_parts(1u32, 4u32, 14u32) {
```

Comment:
This should work for our mainnet and testnet, but won't work for e.g. nctl networks being spun up from version 1.0.0.

Maybe we should try first using `Key::EraSummary` and fall back to `Key::EraInfo` if that fails?

Reply:
Added the change